package.json

```diff
@@ -33,8 +33,8 @@
     "tether": "^1.1.1"
   },
   "peerDependencies": {
-    "react": "^0.14.0",
-    "react-dom": "^0.14.0"
+    "react": "^15.0.1",
```

Comment:
Maybe this should be `^15.0.0` instead, I don't think this should have a preference for patch version

Also, should probably consider being able to support both 0.14 and 15.0, not sure if @souporserious has a preference?